kitten-scientists.js

```diff
@@ -665,9 +785,6 @@ optionsListElement.append(getToggle('festival', 'Festival'));
 // add donation address to bottom of list
 var donate = $('<li/>').append($('<a/>', {
     href: 'bitcoin:' + address + '?amount=0.005&label=Kittens Donation',
-    text: address
-})).prepend($('<img/>', {
-    css: {height: '15px', width: '15px', padding: '3px 4px 0 4px', verticalAlign: 'bottom'},
```

Comment:
This change removes the item since it no longer has a width and height to fill out its contents.